src/path.rs

```diff
@@ -31,6 +33,18 @@ impl PathString for serde_json::Value {
             _ => Cow::Owned(a.to_string()),
         })
     }
+
+    fn evaluate(&self, filter: &Filter) -> Option<async_graphql::Value> {
```

Comment:
Move this function to `impl Mustache` Should take in a `async_graphql::Value` and produce as `async_graphql::Value`